src/box/iproto.cc

```diff
@@ -185,9 +185,7 @@ struct iproto_thread {
 	 * Iproto thread id
 	 */
 	uint32_t id;
-	/*
-	 * Iproto binary listener
-	 */
+	/** Array of iproto binary listeners */
```

Comment:
Please, extract `evio_service` turn into an array into a separate commit. Otherwise you are mixing refactoring (`evio_service`) with a public feature (multilisten). It is hard to tell which changes in this commit belong to what.